### PR TITLE
collect-info.sh: collect usb info

### PR DESCRIPTION
**Pull request by christoph-zededa**
collect info about usb devices to help debugging

### CODE REVIEW

**Review by rouming — APPROVED** [review]
Ack. Please increase version number in the script.

**Review by eriknordmark — COMMENTED** [review]
The eden tests seem to all be failing. Looked at one of them and it has:
**./shell-scripts/collect-info-console.sh: 20: echo: echo: I/O error**

File: pkg/debug/scripts/collect-info.sh [hunk 1]
```diff
@@ -225,6 +225,13 @@ echo "- lsusb, dmesg, ps, lspci, lsblk, lshw, lsof, lsmod, logread, dmidecode, l
 chroot /hostfs lsusb -vvv    > "$DIR/lsusb-vvv"
 chroot /hostfs lsusb -vvv -t > "$DIR/lsusb-vvv-t"
 
+{
+    find /sys/devices/ -name uevent | grep -E '/usb[0-9]/' | xargs grep ""
```
Comment (rene) [inline pkg/debug/scripts/collect-info.sh:229]: the oneline command is the one that you need?
```sh
find /sys/devices/ -path '*/usb[0-9]/*' -name "uevent" -exec awk '{print FILENAME ":" $0}' {} \;
```

**Comment by christoph-zededa** [comment]
> The eden tests seem to all be failing. Looked at one of them and it has: **./shell-scripts/collect-info-console.sh: 20: echo: echo: I/O error**

seems it cannot reach the VM at all:
```
time="2023-11-03T19:07:41Z" level=info msg="Attempt to re-establish connection (17) of (20)"
time="2023-11-03T19:07:46Z" level=info msg="Attempt to re-establish connection (18) of (20)"
time="2023-11-03T19:07:51Z" level=info msg="Attempt to re-establish connection (19) of (20)"
time="2023-11-03T19:07:56Z" level=fatal msg="unable to send request: all connection attempts failed"
ssh: connect to host 127.0.0.1 port 2222: Connection refused
time="2023-11-03T19:08:01Z" level=fatal msg="command ssh failed: exit status 255"
Failed to parse eve-info tarball filename
telnet: Unable to connect to remote host: Connection refused
Trying ::1...
Trying 127.0.0.1...
./shell-scripts/collect-info-console.sh: 20: echo: echo: I/O error
telnet: Unable to connect to remote host: Connection refused
Trying ::1...
Trying 127.0.0.1...
./shell-scripts/collect-info-console.sh: 20: echo: echo: I/O error
telnet: Unable to connect to remote host: Connection refused
Trying ::1...
Trying 127.0.0.1...
./shell-scripts/collect-info-console.sh: 20: echo: echo: I/O error
telnet: Unable to connect to remote host: Connection refused
```

**Review by rouming — APPROVED** [review]
Approve to run eden.

**Comment by christoph-zededa** [comment]
> Approve to run eden.

same issue again ...

**Review by rouming — APPROVED** [review]
Re-approve to run eden.

**Review by rouming — APPROVED** [review]
Re-approve to run eden

**Pull request merged**